### ISSUE TITLE
compilation error is unclear

### ISSUE THREAD

**Issue by ekeren** [winglang/wing#1011]
### I tried this

I tried compiling the following wing application:

```ts (wing)
print("hello".txt);
```

### Expected

```sh
Compilation failed with 1 errors
Error at <filename>:1:1  | property "txt" does not exists on type '"hello"'
```

### Instead, this happened

```sh
Compilation failed with 1 errors
Error | "str" in object.txt does not resolve to a class instance, resource object or enum type
```

### Component

Compiler

### Environment

```markdown
- Wing Version: 0.4.24
- OS:
- Node version:
```

**Comment by staycoolcall911** [comment]
This should be fixed by #172. We should have a uniform error handling for non existing methods.

**Comment by staycoolcall911** [comment]
Done
